easybuild/easyconfigs/i/IOzone/IOzone-3_434-foss-2016a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'IOzone'
+version = '3_434'
+
+homepage = 'http://www.iozone.org/'
+description = """
+IOzone is a filesystem benchmark tool. The benchmark generates and measures a variety of file operations. Iozone has been ported
+ to many machines and runs under many operating systems. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(namelower)s%(version)s.tar']
```

Comment:
please use `['%(namelower)s%(version_major)s_%(version_minor)s.tar']` after changing the version